ambry-router/src/test/java/com.github.ambry.router/PutManagerTest.java

```diff
@@ -878,6 +879,17 @@ public long getSize() {
     return returnedFuture;
   }
 
+  @Override
+  public void setDigestAlgorithm(String digestAlgorithm)
```

Comment:
how did you decide between throwing vs. silently ignoring (which is what you do in `ReadableStreamChannelInputStreamTest`)?